.gitignore

```diff
@@ -16,6 +16,6 @@ lib64
 db.sqlite3
 *.pyc
 pyenv
-myvenv
+venv
```

Comment:
If you want your own folder to be ignore then add it in new line, you changed the folder name, what if other person is using the `myvenv` virtual env, then without him knowing he will add those files too